docs/app/pages/components/components-sections/tree-view/tree-view-custom-node/snippets/app.snippet.html

```diff
@@ -15,6 +15,7 @@
                 <div class="node-wrapper">
 
                     <ux-checkbox
+                      class="m-r-ms"
```

Comment:
Similar to the tree grid, please update the stylesheet so that this works out of the box.